### PR TITLE
Refactor seed.js

### PR DESCRIPTION
**Pull request by KimarieHB**
to be more modular and randomly assign one or two soundtracks per bundle

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+const morgan = require('morgan');
+const parser = require('body-parser');
+// const mongoose = require('mongoose');
```
Comment (gektin) [inline server/index.js]: Probably makes sense to take this out until you write something that requires `mongoose` here.

File: database/scripts/seed.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      track_number: number,
+      name: faker.lorem.words(),
+      song: faker.internet.url()
+    }
+    songs.push(song);
+  }
+  return songs;
+}
+
+for (let i = 1; i <= 10; i++) {
```
Comment (gektin) [inline database/scripts/seed.js]: This `10` seems to be a typo? The `i` here directly seeds the `bundle_id` key in your database (line 43 below), so I would expect this loop to reach 100. 

If, on the other hand, you plan to only have 10 total bonuses and then mix-and-match them from bundle to bundle, I'd rename the `bundle_id` key to something else.

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+})
+
+app.use(morgan('dev'));
+app.use(parser.urlencoded({ extended: true}));
+app.use(parser.json());
+
+// Server test
+app.use('/test', (req, res) => {
+  res.send('3-2-1 testing! Server is serving!');
+})
```
Comment (gektin) [inline server/index.js]: As with your other PR, while I appreciate being able to check this in my browser, a more complete test suite would be useful.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "type": "git",
+    "url": "git+https://github.com/KimarieHB/kimarie-bonuses.git"
+  },
+  "author": "KimarieHB",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/KimarieHB/kimarie-bonuses/issues"
+  },
+  "homepage": "https://github.com/KimarieHB/kimarie-bonuses#readme",
+  "dependencies": {
```
Comment (gektin) [inline package.json:21]: Same comment as before re listing nodemon here. Whatever you end up doing in your bonuses repo should be mirrored here or vice versa.

File: database/models/soundtracks.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+          song: String
+        }
+      ]
+    }
+  ]
+});
+
+const Soundtrack = mongoose.model('Soundtrack', soundSchema);
+
+module.exports = Soundtrack;
```
Comment (gektin) [inline database/models/soundtracks.js]: I believe the airbnb style guide requires a new line at the end of files. Two of the files in this PR have a new line and two don't. Either way, you should pick one and stick with it.

File: database/scripts/seed.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  let additionalBonus = Math.floor(Math.random() * Math.floor(100));
```
Comment (gektin) [inline database/scripts/seed.js]: I don't think you need to `Math.floor(100)` if you're hardcoding `100` in. I also don't think you need `100` given what you're doing with `additionalBonus`, something like `Math.round(Math.random())` will give the same results without raising questions about where the 100 is coming from.

**Review by gektin — COMMENTED** [review]
Running your code seems to work, but a robust test suite would be helpful to confirm that what I see can be believed. 

Otherwise, I think your approach makes sense here, and I think the simple odd/even random number to determine whether there's a second bonus works. However, this code only seeds 10 albums, which will require some changes if the intent is to mix-and-match them.

File: database/scripts/seed.js (hunk 2 above)
Comment (KimarieHB) [reply database/scripts/seed.js]: I forgot to update from my test number to actual bundle number (100).

File: package.json (hunk 4 above)
Comment (KimarieHB) [reply package.json:21]: Added it to `package.json` `"devdependencies"`

File: database/models/soundtracks.js (hunk 5 above)
Comment (KimarieHB) [reply database/models/soundtracks.js]: Committing to a new line at the end of the file always. Nice catch.

File: database/scripts/seed.js (hunk 6 above)
Comment (KimarieHB) [reply database/scripts/seed.js]: 1) The second Math.floor is a typo. Will update.
2) I prefer the distribution of 0s and 1s that I get having a number in there. It's a pattern with a higher velocity.
3) I like the number 100.

File: server/index.js (hunk 1 above)
Comment (KimarieHB) [reply server/index.js]: Removed.

**Review by gektin — APPROVED** [review]
These changes are responsive. I have nothing further. Thanks.

File: database/models/games.js [hunk 7]
```diff
@@ -1,14 +0,0 @@
...
-  game_info: [
-    {
-      title: String,
-      publisher: String
-    }
-  ]
-})
-
-const Game = mongoose.model('Game', gameSchema);
-module.exports = Game;
```
Comment (KimarieHB) [inline database/models/games.js]: I removed this entirely but kept the file incase I get around to adding Game bonuses

File: database/models/soundtracks.js [hunk 8]
```diff
@@ -1,23 +1,23 @@
...
+      ]
     }
   ]
 });
 
 const Soundtrack = mongoose.model('Soundtrack', soundSchema);
 
-module.exports = Soundtrack;
\ No newline at end of file
+module.exports = Soundtrack;
```
Comment (KimarieHB) [inline database/models/soundtracks.js:23]: Moved cover to appear before the tracklist array for readability and add a new line after the last line.

File: database/scripts/seed.js [hunk 9]
```diff
@@ -1,7 +1,6 @@
 const mongoose = require('mongoose');
 const faker = require('faker');
 const Soundtrack = require('../models/soundtracks.js');
-//const GameModel = require('../models/games.js');
```
Comment (KimarieHB) [inline database/scripts/seed.js]: Removed cruft

File: database/scripts/seed.js [hunk 10]
```diff
@@ -14,51 +13,61 @@ mongoose.connect('mongodb://localhost/bonuses', {
   console.log('DB connection error:', err);
 });
 
-const soundtrackSeeds = () => {
```
Comment (KimarieHB) [inline database/scripts/seed.js]: Removed soundtrack seeding from a function making albums global and put the songs generation in it's own helper function to facilitate the random addition of another album to some bonuses.

File: database/scripts/seed.js [hunk 11]
```diff
@@ -14,51 +13,61 @@ mongoose.connect('mongodb://localhost/bonuses', {
...
-        let number = j.toString();
+    let song = {
+      track_number: number,
+      name: faker.lorem.words(),
+      song: faker.internet.url()
+    }
+    songs.push(song);
+  }
+  return songs;
+}
```
Comment (KimarieHB) [inline database/scripts/seed.js:37]: End of songlist generator function

File: package.json [hunk 12]
```diff
@@ -26,6 +26,8 @@
   },
   "devDependencies": {
     "faker": "^5.1.0",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "nodemon": "^2.0.4",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
   }
```
Comment (KimarieHB) [inline package.json:32]: Adde linter back in and nodemon as a devdependency

**Pull request merged**